src/ifcopenshell-python/ifcopenshell/geom/occ_utils.py

```diff
@@ -179,47 +191,48 @@ def set_shape_transparency(ais, t):
 
 def get_bounding_box_center(bbox):
     import OCC.gp
-    
-    bbmin = [0.]*3; bbmax = [0.]*3
+
+    bbmin = [0.] * 3
+    bbmax = [0.] * 3
     bbmin[0], bbmin[1], bbmin[2], bbmax[0], bbmax[1], bbmax[2] = bbox.Get()
-    return OCC.gp.gp_Pnt(*map(lambda xy: (xy[0]+xy[1])/2., zip(bbmin, bbmax)))
+    return OCC.gp.gp_Pnt(*map(lambda xy: (xy[0] + xy[1]) / 2., zip(bbmin, bbmax)))
+
 
-    
 def serialize_shape(shape):
     import OCC.BRepTools
-    
+
     shapes = OCC.BRepTools.BRepTools_ShapeSet()
     shapes.Add(shape)
     return shapes.WriteToString()
-    
+
+
 def create_shape_from_serialization(brep_object):
     import OCC.BRepTools
-    
+
     brep_data, occ_shape, styles = None, None, ()
-    
+
     is_product_shape = True
     try:
         brep_data = brep_object.geometry.brep_data
         styles = brep_object.geometry.surface_styles
-    except: 
-        try: 
+    except BaseException: try:
```

Comment:
this is wrong, I'll hcorrect it, and file an issue with autopep8

Reply:
This was causes by an autopep8 bug:
https://github.com/hhatto/autopep8/issues/320
After updating to the latest autopep8 version, the reformated code is valid.